ocs_ci/framework/conf/default_config.yaml

```diff
@@ -135,6 +135,8 @@ ENV_DATA:
   base_domain: 'qe.rh-ocs.com'
   master_instance_type: 'm4.xlarge'
   worker_instance_type: 'm5.4xlarge'
+  # vm_type_bootstrap is used in AWS UPI deployments
+  vm_type_bootstrap: 'i3.large'
```

Comment:
Is this some recommended one type for bootstrap? IIRC i3 instances are BM with ssd disks.

Reply:
its the default instance type used by OCP QE team